Gruntfile.js

```diff
@@ -40,7 +40,7 @@ module.exports = function (grunt) {
             },
             client: {
                 files: ["client/**/*.js", "spec/**/*.js"],
-                tasks: ["check"],
+                tasks: ["check", "webpack:watch"],
```

Comment:
This isn't quite right - adding webpack:watch here is going to start a watch instance every time the client files are changed. The tasks here are executed once whenever the client files are changed, so you can just run the "build" task directly